cps/main.c

```diff
@@ -1088,9 +1092,19 @@ run_cps(struct net_config *net_conf, struct gk_config *gk_conf,
 		goto stage3;
 	}
 
+	if (gk_conf != NULL) {
+		cps_conf->bgp_max_pkt_burst =
+			RTE_MAX(gk_conf->front_max_pkt_burst,
+				gk_conf->back_max_pkt_burst);
+	}
+
+	if (gt_conf != NULL)
+		cps_conf->bgp_max_pkt_burst = gt_conf->gt_max_pkt_burst;
```

Comment:
Replace `gt_conf->gt_max_pkt_burst` with `RTE_MAX(cps_conf->bgp_max_pkt_burst, gt_conf->gt_max_pkt_burst);` to automatically deal with the case that GK and GT blocks are present at the same time.